### PR TITLE
Backport Predicate Headers

### PR DESCRIPTION
**Pull request by colinschoen**
Backport #8

### CODE REVIEW

**Comment by brunns** [comment]
Maybe backport the complexity fixes too?

**Comment by colinschoen** [comment]
Done!

File: src/mbtest/imposters/stubs.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def as_structure(self):
             response["_behaviors"] = {"wait": self.wait}
         return response
 
+
```
Comment (brunns) [inline src/mbtest/imposters/stubs.py]: This can be reverted.

**Comment by brunns** [comment]
Merged. Anything else you'd like to backport before I cut a release, @colinschoen? Some of the new behaviors, perhaps?

**Comment by colinschoen** [comment]
Yep. I’ll do that after lunch.

**Pull request merged**